fish_prompt.fish

```diff
@@ -619,13 +620,23 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l last_status $status
 
   # Powerline glyphs
-  set -l __bobthefish_branch_glyph            \uE0A0
-  set -l __bobthefish_ln_glyph                \uE0A1
-  set -l __bobthefish_padlock_glyph           \uE0A2
-  set -l __bobthefish_right_black_arrow_glyph \uE0B0
-  set -l __bobthefish_right_arrow_glyph       \uE0B1
-  set -l __bobthefish_left_black_arrow_glyph  \uE0B2
-  set -l __bobthefish_left_arrow_glyph        \uE0B3
+  set -l __bobthefish_branch_glyph            ''
+  set -l __bobthefish_ln_glyph                ''
+  set -l __bobthefish_padlock_glyph           ''
+  set -l __bobthefish_right_black_arrow_glyph ''
+  set -l __bobthefish_right_arrow_glyph       ''
+  set -l __bobthefish_left_black_arrow_glyph  ''
+  set -l __bobthefish_left_arrow_glyph        ''
+
+  if [ "$theme_powerline_fonts" != "no" ]
```

Comment:
I would prefer making powerline fonts the default, as this would have less performance penalties on the prompt at every render.